docs/efun/interactive/send_gmcp.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+title: interactive / send_gmcp
+---
+
+### NAME
+
+    send_gmcp() - sends a message to all living objects
```

Comment:
This sentence seems wrong.

Reply:
oops, used shout() as my template. I'll fix.

Reply:
Fixed in 3a47063